app/src/main/java/org/simple/clinic/summary/PatientSummaryModel.kt

```diff
@@ -40,6 +40,9 @@ data class PatientSummaryModel(
   val isDiabetesManagementEnabled: Boolean
     get() = currentFacility!!.config.diabetesManagementEnabled
 
+  val isTeleconsultationEnabled: Boolean
+    get() = (currentFacility!!.config.teleconsultationEnabled == true)
```

Comment:
The outer parenthesis seems unnecessary here.

Reply:
I mainly placed for readability. Will remove it